src/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,20 @@
+import NextAuth from "next-auth"
+import GoogleProvider from "next-auth/providers/google"
+
+declare global {
+    namespace NodeJS {
+        interface ProcessEnv {
```

Comment:
Again, great job. I never do this. I need to start integrating this into my common practice.